binarySearch/binarySearch.js

```diff
@@ -5,11 +5,10 @@
  * https://en.wikipedia.org/wiki/Binary_search_algorithm
  * */
 
-/**
- *  * const index = binarySearch([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 2);
- *	* console.log(index); // 1
-**/
-
 const binarySearch = (nums, target) => {
-
-};
\ No newline at end of file
+    for (let i = 0; i < nums.length; i++) {
+        if (nums[i] === target) return i;
```

Comment:
So this search isn't binary. Binary would be an algorithm that cuts your data in half, searches that piece, and then moves on based on whether or not its found. check out the links provided.